docs/lifecycle/README.md

```diff
@@ -0,0 +1,50 @@
+# Set up Object Lifecycle Configuration [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
+
+Using object lifecycle configuration on buckets to setup automaic objects deletion after a specified days or a specified date.
+
+## 1. Prerequisites
+- Install MinIO - [MinIO Quickstart Guide](https://docs.min.io/docs/minio-quickstart-guide).
+- Install AWS Cli - [Installing the AWS CLI - AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+
+## 2. Setup a bucket lifecycle using AWS
+
+1. Create a JSON file which contains the bucket lifecycle configuration, as the following example:
```

Comment:
```suggestion
1. Create the following bucket lifecycle configuration expires the objects under the prefix `uploads/2015` on 2020-01-01T00:00:00.000Z since upload and the objects under `temporary-uploads/` after 7 days after upload, as shown below:
```